Tests/test_cellosaurus.py

```diff
@@ -63,19 +63,23 @@ def test_parse(self):
             self.assertEqual(record["ST"][7], "TH01: 7")
             self.assertEqual(record["ST"][8], "TPOX: 8,11")
             self.assertEqual(record["ST"][9], "vWA: 14,16")
-            self.assertEqual(record["DI"][0], "NCIt; C3965; Xeroderma pigmentosum,"
-                                              " complementation group A")
+            self.assertEqual(
+                record["DI"][0],
+                "NCIt; C3965; Xeroderma pigmentosum, complementation group A",
+            )
             self.assertEqual(record["OX"][0], "NCBI_TaxID=9606; ! Homo sapiens")
             self.assertEqual(record["SX"], "Female")
             self.assertEqual(record["CA"], "Finite cell line")
             record = next(records)
             self.assertEqual(record["ID"], "1-5c-4")
             self.assertEqual(record["AC"], "CVCL_2260")
-            self.assertEqual(record["SY"],
-                             "Clone 1-5c-4; Clone 1-5c-4 WKD of "
-                             "Chang Conjunctiva; "
-                             "Wong-Kilbourne derivative of "
-                             "Chang conjunctiva; ChWK")
+            self.assertEqual(
+                record["SY"],
+                "Clone 1-5c-4; Clone 1-5c-4 WKD of "
+                "Chang Conjunctiva; "
+                "Wong-Kilbourne derivative of "
+                "Chang conjunctiva; ChWK",
+            )
```

Comment:
I would suggest splitting that long string into just two lines (at the semi colon), rather than four lines?

Reply:
Yes, done.